crates/chain-config/src/config/state.rs

```diff
@@ -57,5 +57,5 @@ pub trait ChainConfigDb {
     /// Returns *all* unspent message configs available in the database.
     fn get_message_config(&self) -> StorageResult<Option<Vec<MessageConfig>>>;
     /// Returns the last available block height.
-    fn get_block_height(&self) -> StorageResult<Option<BlockHeight>>;
+    fn get_block_height(&self) -> StorageResult<BlockHeight>;
```

Comment:
Why is the option removed? If the chain is empty won't this be none?

Reply:
All chains have a genesis block now, so it should not be `None` + we have `NotFound` to represent `None` case.